src/information-panel/information-panel.scss

```diff
@@ -187,3 +192,34 @@
     top: $iui-m * -0.5;
   }
 }
+
+/// Labels and inputs aligned using css grid
+@mixin iui-information-body-content {
+  .iui-input-label {
+    font-weight: normal;
```

Comment:
Don't we have a variable for that? Because regular is `400`.

Reply:
what's the benefit?

Reply:
In case we decide to change it.

Reply:
I agree with Vilius here.

Also, I read somewhere (can't find the article again) that you should use numbers instead of values like `bolder` because `bolder` may be different across devices / browsers.

Reply:
> Also, I read somewhere (can't find the article again) that you should use numbers instead of values like bolder because bolder may be different across devices / browsers.

I have not been able to find a source for that.

But changed